vizro-ai/examples/dashboard_ui/actions.py

```diff
@@ -13,16 +15,50 @@
 from vizro.models.types import capture
 from vizro_ai import VizroAI
 
+try:
+    from langchain_anthropic import ChatAnthropic
+except ImportError:
+    ChatAnthropic = None
+
+try:
+    from langchain_mistralai import ChatMistralAI
+except ImportError:
+    ChatMistralAI = None
+
 logger = logging.getLogger(__name__)
 logger.setLevel(logging.INFO)  # TODO: remove manual setting and make centrally controlled
 
-SUPPORTED_VENDORS = {"OpenAI": ChatOpenAI}
+SUPPORTED_VENDORS = {"OpenAI": ChatOpenAI, "Anthropic": ChatAnthropic, "Mistral": ChatMistralAI}
+
+SUPPORTED_MODELS = {
+    "OpenAI": [
+        "gpt-4o-mini",
+        "gpt-4o",
+        "gpt-4",
+        "gpt-4-turbo",
+        "gpt-3.5-turbo",
+    ],
+    "Anthropic": [
+        "claude-3-5-sonnet-20240620",
+        "claude-3-opus-20240229",
+        "claude-3-sonnet-20240229",
+        "claude-3-haiku-20240307",
```

Comment:
```suggestion
        "claude-3-5-sonnet-latest",
        "claude-3-opus-latest",
        "claude-3-sonnet-20240229",
        "claude-3-haiku-20240307",
```

Reply:
It would be easier to maintain by pointing to the most recent snapshot of a given model. whenever possible

Reply:
Yes, let's do that. Can you also try if the latest works for sonnet and haiku? I suspect it will. For bonus points, could you quickly update this in `_llm_models.py` and the docs too (sorry for hijacking the PR, but this should take 5min?)

Reply:
Sure, I've updated the `_llm_models.py` and the docs. The "claude-3-sonnet-20240229" and "claude-3-haiku-20240307" do not work with "latest".